example/app.py

```
@@ -5,12 +5,23 @@
+- /memory : Endpoint that consumes high memory
+- /cpu : Endpoint that spikes CPU usage
+- /leak : Endpoint that creates memory leaks
+- /crash : Endpoint that may crash the worker
+- /oom : Endpoint that may trigger OOM killer
+- /stress : Endpoint that combines multiple stressors
 """
 
+import gc
+import os
 import random
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)